app/controllers/api/v1/authors_controller.rb

```diff
@@ -0,0 +1,48 @@
+module Api
+  module V1
+    class AuthorsController < ApplicationController
+      before_action :set_author, only: [:show, :update, :destroy]
+
+      # GET /authors
+      def index
+        authors = Author.all
+        respond_with authors, class: SerializableAuthor
```

Comment:
no need for the extra variable.

Also, just extract the `SerializableAuthor` constant to a private method called `serializer` and use it everywhere.